Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1179,11 +1179,9 @@ def get_remote_data_command(service, args, close_incident):
 
     for item in results.ResultsReader(service.jobs.oneshot(search)):
         updated_notable = parse_notable(item, to_dict=True)
-    delta = {field: updated_notable.get(field) for field in INCOMING_MIRRORED_FIELDS if updated_notable.get(field)}
-
-    if delta:
-        demisto.debug('notable {} delta: {}'.format(notable_id, delta))
-        if delta.get('status') == '5' and close_incident:
+    if updated_notable:
```

Comment:
from the original code, it looks like we don't need to check it.
was made intentionality?